tests/cases/fourslash/importNameCodeFixUMDGlobalReact2.ts

```diff
@@ -17,5 +17,8 @@
 ////[|<div/>|]
 
 goTo.file("/a.tsx");
-verify.not
-verify.importFixAtPosition([]);
+verify.importFixAtPosition([
+`import { factory } from "./factory";
```

Comment:
I assume that this suggestion is correct?

Reply:
Yeah -- it imports the `export function factory` that was defined in `factory.ts`.